packages/route/src/Application/ResourceLocator/ResourceLocatorGenerator.php

```diff
@@ -22,6 +22,7 @@
     public function __construct(
         private RouteRepositoryInterface $routeRepository,
         private PathCleanupInterface $pathCleanup,
+        private ?RouteSchemaProcessor $routeSchemaProcessor = null,
```

Comment:
fine to force this as a bugfix:


```suggestion
        private RouteSchemaProcessorInterface $routeSchemaProcessor,
```